velox/dwio/parquet/writer/Writer.h

```diff
@@ -36,7 +36,12 @@ struct WriterOptions {
   int32_t rowsInRowGroup = 10'000;
   int64_t maxRowGroupLength = 1'024 * 1'024;
   int64_t dictionaryPageSizeLimit = 1'024 * 1'024;
-  double bufferGrowRatio = 1;
+  // Parquet writer need to write the output into ArrowDataBufferSink.
+  // bufferGrowRatio parameter determines the growth factor used when invoking
+  // the reserve method of ArrowDataBufferSink, thereby helping to minimize
+  // frequent memcpy operations. And the default value 1.5 refer to the
+  // folly/FBVector's grow factor.
```

Comment:
Is this a reference to https://github.com/facebook/folly/blob/main/folly/FBVector.h#L1142 ? It is a bit surprising since the code doesn't use FBVector, I believe.

Reply:
Yes. I have experimented with values of 1.5, 2, 3, and 4, and observed that the performance is nearly identical across these values.